lib/virtualGraphs.js

```diff
@@ -9,7 +9,8 @@ const list = conn => {
   }).then(httpBody);
 };
 
-const add = (conn, name, mappings, options) => {
+const add = (conn, name, mappings, options, meta) => {
+  const { db } = meta;
```

Comment:
This will throw if `meta` is undefined, as it will be for most existing calls to this method, and I don't think we want to make it required (right?). So, suggestion:

```suggestion
const add = (conn, name, mappings, options, meta = {}) => {
  const { db } = meta;
```